README.md

```diff
@@ -20,13 +20,23 @@ Please refer to [the preCICE documentation](https://www.precice.org/installation
 
 ## Installing the package
 
-We recommend using pip3 (version 19.0.0 or newer required) for the sake of simplicity. You can check your pip3 version via `pip3 --version`. To update pip3, use the following line:
+Generally, it is recommended to work in a virtual environment when using the preCICE Python bindings. For this purpose, you might have to run `apt install python3-venv` first.
+
+Before running any of the commands below, please create a virtual environment at a location of your choice by running:
```

Comment:
```suggestion
Create a virtual environment in your working directory by running:
```
I would suggest sticking to imperative mood as we do in most of our documentation.